test/mocha/10-api.js

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ */
+const bedrock = require('bedrock');
```

Comment:
for all the files that switched from ESM to CJS (import to require), please add `'use strict';` at the top.